public/styles.css

```diff
@@ -0,0 +1,43 @@
+.navTopMargin {
+  margin-top: 60px;
+}
+.pageTitle {
```

Comment:
Formatting inconsistency: Some classes are separated by newline and some aren't, and it isn't clear why